run.sh

```diff
@@ -2,6 +2,39 @@
 
 set -ex
 
+# Ensure we're running on a Raspberry Pi
+if ! grep -q "Raspberry Pi" /proc/device-tree/model 2>/dev/null; then
+	echo "This script is intended to run on a Raspberry Pi."
+	exit 1
+fi
+
+# Ensure symbolic link is created (only if it doesn't exist)
```

Comment:
1. why link this?
2. will this fail if the device already has rust things installed when you install the module?

Reply:
The failure mode I was seeing was that /root/.local wasn't linked to /root/.cargo, I can't tell you why that's a problem but addressing what the terminal is telling me to address.

Reply:
2. mebbe, should I just call it quits and try pyinstaller again? I'll only be able to develop on a pi because of the `rpi.gpio` library used in this module, which I guess isn't that bad.

Reply:
your call, pros + cons. guessing we can get this working without the symlink

Reply:
oh wait this is my mistake -- looks like the uv install script installs in ~/.local/bin, not ~/.cargo/bin

either this changed recently, or I always had it wrong